src/app/pages/home/home.js

```diff
@@ -52,16 +52,12 @@ export default class Home extends BaseView {
             return;
         }
 
-        let secondaryNavHeight = appView.header.secondaryNavHeight;
-
-        if (window.pageYOffset > secondaryNavHeight && !appView.header.isPinned()) {
-            let height = appView.header.height;
-
-            appView.header.reset().collapse().pin();
-            this.el.style.paddingTop = `${height / 10}rem`;
-        } else if (window.pageYOffset <= secondaryNavHeight && !appView.header.isTransparent()) {
-            appView.header.reset().transparent();
-            this.el.style.paddingTop = '0';
+        if (window.pageYOffset >= 250) {
+            appView.header.pin().visible();
+        } else if (window.pageYOffset <= 249 && window.pageYOffset >= 150) {
```

Comment:
Shouldn't that be dynamically calculated though?  If you want it to slide in below the height of the header, then you could always do something like `appView.header.secondaryNavHeight + 20;`.